57.md

```diff
@@ -168,16 +168,20 @@ A client can retrieve `zap receipt`s on events and pubkeys using a NIP-01 filter
 
 ### Appendix G: `zap` tag on zapped event
 
-When an event includes a `zap` tag, clients SHOULD calculate the lnurl pay request based on it's value instead of the profile's field. An optional third argument on the tag specifies the type of value, either `lud06` or `lud16`.
+When an event includes one or more `zap` tags, clients SHOULD calculate the lnurl pay request based on their value instead of the profile's field. The tag's second argument is a `lud06` address, `lud16` identifier or `hex` string of the receiver's pub key. An optional third argument specifies the type of the previous value: `lud06`, `lud16` or `pubkey`. An optional fourth parameter specifies the weight (a generalization of a percentage) assigned to the respective receiver. Clients should parse all weights, calculate a sum, and then a percentage to each member. If weights are not present, CLIENTS should equally divide the zap amount to all zap receivers. If weights are only partially present, receivers without a weight should not be zapped (`weight = 0`).
 
 ```json
 {
     "tags": [
-        [ "zap", "pablo@f7z.io", "lud16" ]
+        [ "zap", "LNURL..", "lud06", 10 ],  // 10%
```

Comment:
LGTM but the number must be a string as tags can only contain strings
```suggestion
        [ "zap", "LNURL..", "lud06", "10" ],  // 10%
```

Reply:
Good catch @Cameri I just fixed this.